site/_posts/2022-03-10-announcing-fusionauth-1-35.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.35
+description: This release includes bug fixes, internal updates, and support for API calls in FusionAuth lambdas.
+author: Dan Moore
+image: blogs/release-1-35/product-update-fusionauth-1-35.png
+category: blog
+tags: tbd topic-troubleshooting feature-advanced-threat-detection
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.35 of FusionAuth. This version shipped Mar 9, 2022. 1.35 includes bug fixes, internal updates, and support for API calls in FusionAuth lambdas.
```

Comment:
That is one way to say it, but you can call anything, it doesn't have to be an API. I would just say "support for HTTP requests in FusionAuth lambdas."